### DISCUSSION TITLE
My git copilot application was not accepted.

### DISCUSSION THREAD

**Discussion by orarange** [community/community#10392] [Copilot Conversations] [answered]
I'm sorry to bother you.
I applied for git copilot about 3 months ago, but it hasn't been approved yet.
I'm very sorry, but I'd like to speed up the approval process.
Thank you for your time.

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
